java/rocksjni/portal.h

```diff
@@ -2477,6 +2478,35 @@ class JniUtil {
     return op(key_slice);
   }
 
+  /*
+   * Helper for operations on a key on java byte array
+   * Copies values from jbyte array to slice and performs op on the slice.
+   * for example WriteBatch->Delete
+   *
+   * TODO(AR) could be extended to cover returning ROCKSDB_NAMESPACE::Status
```

Comment:
Why does this have my initials against the TODO?

Reply:
Also the naming seems odd as this has nothing to do with an Indirect ByteBuffer (instead it appears to work with a byte array)

Reply:
indirect byte buffer & byte array share the same interface.